README.md

```diff
@@ -1,33 +1,42 @@
 
-We have an example code for .NET available at [https://github.com/NordPool/public-intraday-api-net-example](https://github.com/NordPool/public-intraday-api-net-example). 
+This sample application uses Java data objects published in [Java Intraday API package](https://maven.pkg.github.com/NordPool/public-intraday-api)
+
+There is also .NET sample application at [https://github.com/NordPool/public-intraday-api-net-example](https://github.com/NordPool/public-intraday-api-net-example). 
 
 ## Disclaimer ##
 
 We offer the data transfer object code and client code examples to aid the development against Nord Pool's API at no warranty whatsoever. Clients are solely responsible for separately testing and ensuring that interaction with Nord Pool works according to their own standards.
 
 ## Building ##
 
-Before you begin, you should have the library  [https://github.com/NordPool/public-intraday-api](https://github.com/NordPool/public-intraday-api) to be available through GitHub packages. 
+Before you begin, you should have the library [https://github.com/NordPool/public-intraday-api](https://github.com/NordPool/public-intraday-api) to be available through GitHub packages. 
+For more detailed instruction see **Authenticating to GitHub Packages** section of this document.
+
 
-Then you may import the code in your favorite IDE and execute **gradle build**.
+Then you may import the code in your favorite IDE and build project with **gradle build** command.
 
-In order to connect, **you must edit the src\main\resources\example.properties** file:
+All the relevant variables for connecting are located in [application.properties](/src/main/resources/application.properties) and Gradle's [build.gradle](build.gradle) configuration files. 
+Before running the example, user credentials should be updated to [application.properties](/src/main/resources/application.properties):
 ```
 #!
-sso.user=your_user
-sso.password=your_password
+credentials.userName=your_user
+credentials.password=your_password
 ```
+
 These credentials shall be obtained from [idapi@nordpoolgroup.com](mailto:idapi@nordpoolgroup.com) prior to running the example.
 
-Additionally, make sure that all the properties in the file point to correct addresses.
-Finally, either run the program using **gradle bootRun** or run the class PublicApiApplication.java.
+Additionally, make sure that all the other variables in the [application.properties](/src/main/resources/application.properties) file point to correct addresses.
+Finally, run the solution with **gradle run** command or by run a startup class [ApplicationMain.java](/src/main/java/nps/id/publicapi/java/client/ApplicationMain.java) with your Java IDE.
+
+The program will create two parallel connections that targets both: **Market data** web service and old **Trading** web service.
```

Comment:
> and old **Trading** web service


'old' will sound confusing for a new clients let's not use 'old' term.
 that targets both: **Market data** web service and **Trading** web service

Reply:
Yes i forgot to remove this while was updating names of web services